test/regression/cases/file_to_blackhole_0ms_latency_http1/experiment.yaml

```diff
@@ -0,0 +1,39 @@
+optimization_goal: egress_throughput
+erratic: false
+
+target:
+  name: datadog-agent
+  command: /bin/entrypoint.sh
+  cpu_allotment: 8
+  memory_allotment: 30g
```

Comment:
Would you set this to something like 1.5GiB? We're asserting that the memory doesn't blow past 1.2GiB below.